src/components/TranslationsTab.tsx

```diff
@@ -116,7 +116,7 @@ const TranslationTab = (props: TranslationTabProps) => {
     return (
       <ThemeProvider>
         <Box padding={4}>
-          <Layer zOffset={1000}>
```

Comment:
I have no problem with this going away, just making a comment for myself here in case we need to revert to it in the future.